src/nanovg_gl.h

```diff
@@ -345,12 +346,12 @@ static void glnvg__blendFuncSeparate(GLNVGcontext* gl, const GLNVGblend* blend)
 
 static GLNVGtexture* glnvg__allocTexture(GLNVGcontext* gl)
 {
-	GLNVGtexture* tex = NULL;
-	int i;
+ 	GLNVGtexture* tex = NULL;
+	int i, idx = -1, salt;
 
 	for (i = 0; i < gl->ntextures; i++) {
 		if (gl->textures[i].id == 0) {
-			tex = &gl->textures[i];
+			idx = i;
 			break;
 		}
 	}
```

Comment:
``tex == NULL`` below this line should probably be ``idx == -1`` now?

Reply:
That's correct, below should be:
```C
if (idx == -1) {
```